lib/client/mediator-subscribers/read.js

```diff
@@ -33,6 +35,8 @@ module.exports = function readResultSubscriber(resultEntityTopics, resultClient)
 
     resultClient.manager.read(parameters.id)
     .then(function(result) {
+      SyncEvents.updateSyncStatus(result);
```

Comment:
This should return the status and append it to the result value.

Might also want to call it `getSyncStatus`